Video_analyser_code/VideoAnalyser.py

```diff
@@ -75,47 +75,37 @@ def define_regions(self):
     def define_regions(self):
         # Define the regions of interest (ROI) for each mouse and their specific zones
         regions = {
-            'm1_c': [(500, 110), (535, 180)],  # Mouse 1 Cooperate Zone (Top Right)
-            'm1_cen': [(610, 280), (680, 350)],  # Mouse 1 Center Zone (Center Right)
-            'm1_d': [(500, 400), (535, 470)],  # Adjusted Mouse 1 Defect Zone (Bottom Right)
-
-            'm2_c': [(455, 110), (490, 180)],  # Mouse 2 Cooperate Zone (Top Left)
-            'm2_cen': [(330, 280), (400, 350)],  # Mouse 2 Center Zone (Center Left)
-            'm2_d': [(455, 400), (490, 470)],  # Mouse 2 Defect Zone (Bottom Left)
-            'experimenter_start_zone': [(440, 280), (520, 350)]
+            'm1_c': [(455, 120), (490, 190)],  # Mouse 2 Cooperate Zone (Top Left)
+            'm1_cen': [(330, 280), (400, 350)],  # Mouse 2 Center Zone (Center Left)
+            'm1_d': [(455, 420), (490, 490)],  # Mouse 2 Defect Zone (Bottom Left)
+            'm2_c': [(510, 120), (545, 190)],  # Mouse 1 Cooperate Zone (Top Right)
+            'm2_cen': [(610, 280), (680, 350)],  # Mouse 1 Center Zone (Center Right)
+            'm2_d': [(510, 420), (545, 490)],  # Adjusted Mouse 1 Defect Zone (Bottom Right)
+
```

Comment:
I see that the experimenter start zone is still here. Do we want it?

Reply:
no thats no longer needed